core/services/ocr2/plugins/directrequestocr/scripts/gen_oracle_contract_config.go

```diff
@@ -14,6 +16,14 @@ import (
 	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
 )
 
+type OCR2DROracleConfig struct {
+	Signers, Transmitters []string
+	F                     uint8
+	OnchainConfig         string
```

Comment:
In case you didn't know: you can add attributes for json serialization as following:
```
OnchainConfig string `json:"onchain_config,omitempty"`
```
This way you can override the field names in the json and omit the field if it has empty value.
Just FYI.